ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -379,6 +379,9 @@ def get_target_handle_id_by_source_node_id(self, _source_node_id: UUID) -> UUID:
     @cached_property
     def node_id(self) -> UUID:
         """Can be overridden as a class attribute to specify a custom node id."""
+        wrapped_node = get_wrapped_node(self._node)
+        if wrapped_node:
+            return uuid4_from_hash(wrapped_node.__qualname__)
```

Comment:
Instead of editing this method, change the key of `serialized_nodes` to use the wrapped_node's id